src/style.css

```
@@ -1 +1,12 @@
 /* styles go here */
+fieldset .form-field {
+  margin-bottom: 10px;
+}
+
+fieldset > .form-field:last-child {
+  margin-bottom: 0;
+}
+
+fieldset {
+  margin-bottom: 20px;
+}
```

Don't style using tags. Add classes where necessary.
Also, there's no need to increase selector specificity.
```suggestion
.form-field {
  margin-bottom: 10px;
}

.form-field:last-child {
  margin-bottom: 0;
}

addSomeClass {
  margin-bottom: 20px;
}
```